README.md

```diff
@@ -33,7 +33,10 @@ housekeeping:
     password: Ch4ll3ng3
 ```
 
-Note: To use MySQL and similar database systems, the schema specified in the configuration needs to exist, as the value for `housekeeping.data-source.url` needs to be a valid URI.
+Notes:
+
+* To use MySQL and similar database systems, the schema specified in the configuration needs to exist, as the value for `housekeeping.data-source.url` needs to be a valid URI.
+* Because Housekeeping is built in Java 7, it would be safe to use the latest MariaDB jar for Java 7 support, `mariadb-java-client-1.8.0.jar`.
```

Comment:
Not sure we need to add this second bullet. Users should look up the Driver needed by the RDS instance they choose to use.

Reply:
Ah, so it doesn't depend on the Houskeeeping JDK version?
I haven't actually tested it with a higher version for the jar, so I should do that.

Reply:
Yeah, let's not mention specific versions here, @patduin 's suggested text from above could go here:

> Some users have seen issues using the MySQL driver with AWS Aurora, AWS recommends to use the MariaDB driver for those use cases.

Reply:
at runtime you can run in higher jdk, so probably a higher one will be ok. The Driver depends on the environment that users run on and we can't account/document all possible variations.

Reply:
Okay, that's changed now :)